sources/packages/backend/libs/integrations/src/institution-integration/ece-integration/ece-response.processing.service.ts

```diff
@@ -312,6 +332,51 @@ export class ECEResponseProcessingService {
     return disbursementProcessingDetails;
   }
 
+  /**
+   * Validate disbursement data.
+   * @param disbursement disbursement.
+   */
+  private validateDisbursement(disbursement: DisbursementDetails): void {
+    const errors: string[] = [];
+
+    const hasInvalidEnrolmentConfirmationFlag = disbursement.awardDetails.some(
+      (award) => !["Y", "N"].includes(award.enrolmentConfirmationFlag),
```

Comment:
"Y" and "N" can be part of a simple enum specific for this integration. The enum can be used here (`Object.values`) and also when checking for the "Y" flag.